### PR TITLE
import: supports also non-output files/dirs

### LINKED ISSUE #835: import: allows to download/copy also files that are not output
Documentation update necessary after PR https://github.com/iterative/dvc/pull/2889 for issue https://github.com/iterative/dvc/issues/2862

I will do this

### PR DESCRIPTION
**Pull request by Baranowski**
This update reflects https://github.com/iterative/dvc/pull/2889

Fix #835

Disregard the recommendations below if you use **Edit on GitHub** button to improve the docs in place.

❗ Please read the guidelines in the [Contributing to the Documentation](https://dvc.org/doc/user-guide/contributing/docs) list if you make any substantial changes to the documentation or JS engine.

🐛 Please make sure to mention `Fix #issue` (if applicable) in the description of the PR. This enables GitHub to link the PR to the corresponding bug and close it automatically when PR is merged.

Thank you for the contribution - we'll try to review and merge it as soon as possible. 🙏

### CODE REVIEW

File: static/docs/command-reference/import.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # import
 
-Download or copy an <abbr>output</abbr> file or directory from any <abbr>DVC
-repository</abbr> (e.g. hosted on GitHub) into the <abbr>workspace</abbr>. DVC
-creates a [DVC-file](/doc/user-guide/dvc-file-format) with information about the
-data source, which can later be used to [update](/doc/command-reference/update)
-the import.
+Download or copy a file or directory, possibly being an <abbr>output</abbr>,
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import.md]:
```suggestion
Download a file or directory, possibly being an <abbr>output</abbr>,
```
Needs rewrapping of course.

File: static/docs/command-reference/import.md [hunk 2]
```diff
@@ -1,10 +1,11 @@
...
+from any <abbr>DVC repository</abbr> (e.g. hosted on GitHub) into the
+<abbr>workspace</abbr>. DVC creates a
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import.md]:
```suggestion
<abbr>workspace</abbr>. This also creates a
```

File: static/docs/command-reference/import.md [hunk 3]
```diff
@@ -32,15 +33,16 @@ source <abbr>project</abbr>. Both HTTP and SSH protocols are supported for
 online repositories (e.g. `[user@]server:project.git`). `url` can also be a
 local file system path to an "offline" repository (in this case instead of
 downloading, DVC may copy the target data from the external source project or
-it's cache).
+its cache).
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import.md]: Good catch. Please also fix in static/docs/command-reference/get.md

File: static/docs/command-reference/import.md [hunk 4]
```diff
@@ -32,15 +33,16 @@ source <abbr>project</abbr>. Both HTTP and SSH protocols are supported for
...
-will not find data files directly in the external Git repository.) The source
-project should have a default [DVC remote](/doc/command-reference/remote)
-configured, containing them.)
+to be downloaded within the source project. It should point to a file or
+directory tracked by that project. It might be an <abbr>output</abbr> – in that
+case it should be specified in one of the
+[DVC-files](/doc/user-guide/dvc-file-format) of the repository at `url` and
+might not be present in the repository itself. When importing an output, the
+source project should have a default [DVC remote](/doc/command-reference/remote)
+configured, containing it.
```
Comment (jorgeorpinel) [inline static/docs/command-reference/import.md]: Just copy the same paragraph in https://dvc.org/doc/command-reference/get instead, please.

**Comment by Baranowski** [comment]
@jorgeorpinel, this is ready to review. All the other changes in `get` docs seems to be related to the "download/obtain" changes.

**Comment by shcheklein** [comment]
@Baranowski could you please resolve the conflicts ^^

File: static/docs/command-reference/import.md (hunk 1 above)
Comment (jorgeorpinel) [reply static/docs/command-reference/import.md]: Oops sorry I also meant to suggest that we can just remove the `, possibly being an <abbr>output</abbr>,` part from here. We want to keep the first paragraph as simple as possible.

Comment (jorgeorpinel) [reply static/docs/command-reference/import.md]: Having `<abbr>output</abbr>` somewhere in the description could be useful though, but would also need to add in the `dvc get` cmd ref... Up to you, not super needed.

Comment (jorgeorpinel) [reply static/docs/command-reference/import.md]: You know what, I'll take care of this post-merge. Thanks @Baranowski!

Comment (jorgeorpinel) [reply static/docs/command-reference/import.md]: Addressed in d99cb390.

Comment (Baranowski) [reply static/docs/command-reference/import.md]: Thank you!

**Pull request merged**